packages/server/api/src/app/ai/mcp/llm-query-router.ts

```diff
@@ -207,6 +207,7 @@ const getSystemPrompt = async (
     `IMPORTANT: Tables tools should always be included in the output if the user asks a question involving those table names: ${openopsTablesNames.join(
       ', ',
     )}. ` +
+    "NOTE: The 'session-sql' tool is ONLY for querying tables created by AWS cost/billing MCP tools and performing temporary analysis within the session. It is NOT for user's database or table related queries. " +
```

Comment:
I don't like this @ravikiranvm .
There might be other tools in the future that cause this problem.
I think a broader solution is needed, to say that certain tools do not mix with each other.
Please consult with Eran.